extra-graphics/graphviz/autobuild/beyond

```diff
@@ -1,3 +1,5 @@
-abinfo "Moving misplaced files under /lib to /usr/lib ..."
-cp -rv "$PKGDIR"/lib/* "$PKGDIR"/usr/lib/
-rm -rv "$PKGDIR"/lib
+if [ -d "$PKGDIR"/lib ]; then
```

Comment:
Or lib64? Does it happen?

Reply:
Well, I think it's files from Tcl, so I think it does not happen, at least on riscv64/amd64.

Reply:
Cool.